saplings/circuits/src/components/circuitDetails/CircuitDetails.scss

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+.table-container {
+  overflow:auto;
+  margin-top: 3rem;
+
+  .nodes-table {
+    width: 100%;
+    border-collapse: collapse;
+
+    td.no-nodes-msg {
+      width: 100%;
+      min-height: 20%;
+      text-align: center;
+      color: var(--color-light-grey);
+      font-size: 2rem;      
+    }
+
+    th, td {
+      border-bottom: 1px solid var(--color-light-grey);
+      padding: 2rem 2rem 2rem 0;
+      max-width: 13rem;
+    }
+
+    .table-header th {
+      color: var(--color-grey);
+      font-weight: 100;
+      font-size: 0.8rem;
+      text-align: left;
+      cursor: pointer;
+      padding-bottom: 1rem;
+      background-color: var(--background-color-white);
+      box-shadow: 0 4px 4px -4px rgba(0,0,0,.25);
+      position: -webkit-sticky;
+      position: sticky;
+      top:0;
+    }
+
+    .table-row {
+      a {
+          text-decoration: none
+      }
+      td {
+        font-size: 0.8rem;
+        vertical-align: middle;
+      }
+    }
+  }
+}
+
+.service-container {
```

Comment:
Should all these rules about "service-container" be included in this commit?

Reply:
Removed